components/mailchimp/sources/new-list-event/new-list-event.js

```diff
@@ -0,0 +1,42 @@
+const common = require("../common/http-based");
+const listEventTypes = require('../list-event-types.js');
```

Comment:
If you change `list-event-types` to `constants`
```suggestion
const common = require("../common/http-based");
//const listEventTypes = require('../list-event-types.js');
const constants = require("../constants.mjs");
// or import constants from "../constants.mjs";
```